exporter/exporterhelper/internal/queuebatch/queue_batch.go

```diff
@@ -65,18 +66,20 @@ func newQueueBatch(
 	if cfg.Batch != nil {
 		if oldBatcher {
 			// If user configures the old batcher we only can support "items" sizer.
-			b = newDefaultBatcher(*cfg.Batch, batcherSettings[request.Request]{
-				sizerType:  request.SizerTypeItems,
-				sizer:      request.NewItemsSizer(),
-				next:       next,
-				maxWorkers: cfg.NumConsumers,
+			b = newMultiBatcher(*cfg.Batch, batcherSettings[request.Request]{
+				sizerType:   request.SizerTypeItems,
+				sizer:       request.NewItemsSizer(),
+				partitioner: nil,
```

Comment:
The old batcher is just a separate configuration. I think we can keep them on par and pass the partitioner here as well